# Rationale:
# 1. Mozilla's code contains a hard-coded check: if a search engine's name
#    is "Yahoo" or "Google", some information is added to the URL to make
#    the search recognizeable as coming from the Sherlock plugin.  I fail
#    to see a reason for it.
# 2. Since the /usr/lib*/mozilla*/searchplugins/ dir is writable only by
#    root, there is no need for update* tags.
--- ./xpfe/components/search/datasets/google.src~	2005-07-07 14:11:24.000000000 +0200
+++ ./xpfe/components/search/datasets/google.src	2005-07-07 14:21:11.000000000 +0200
@@ -1,12 +1,14 @@
 # Mozilla/Google plug-in by amitp+mozilla@google.com
 
+# modified by PLD Team
+
 <search 
-   name="Google"
+   name="google"
    description="Google Search"
    method="GET"
    action="http://www.google.com/search"
-   update="http://www.google.com/mozilla/google.src"
-   updateCheckDays=1   
+#  update="http://www.google.com/mozilla/google.src"
+#  updateCheckDays=1   
    queryEncoding="utf-8"
    queryCharset="utf-8"
 >
--- ./browser/locales/en-US/searchplugins/google.src~	2004-08-29 09:29:59.000000000 +0200
+++ ./browser/locales/en-US/searchplugins/google.src	2005-07-07 14:20:29.000000000 +0200
@@ -1,12 +1,14 @@
 # Mozilla/Google plug-in by amitp+mozilla@google.com
 
+# modified by PLD Team
+
 <search 
-   name="Google"
+   name="google"
    description="Google Search"
    method="GET"
    action="http://www.google.com/search"
-   update="http://www.google.com/mozilla/google.src"
-   updateCheckDays=1   
+#  update="http://www.google.com/mozilla/google.src"
+#  updateCheckDays=1   
    queryEncoding="utf-8"
    queryCharset="utf-8"
 >
--- ./browser/locales/en-US/searchplugins/yahoo.src~	2004-11-03 00:55:39.000000000 +0100
+++ ./browser/locales/en-US/searchplugins/yahoo.src	2005-07-09 12:06:59.000000000 +0200
@@ -5,12 +5,14 @@
 # Last updated August 11, 2004
 
 <SEARCH
-    version = "7.1" 
-   name="Yahoo"
+#   version = "7.1" 
+   name="yahoo"
    description="Yahoo Search"
    searchForm="http://www.yahoo.com/r/so"
    method="GET"
    action="http://search.yahoo.com/search" 
+   queryEncoding="utf-8"
+   queryCharset="utf-8"
 >
 
 <input name="p" user>
@@ -34,9 +36,9 @@
 
 <browser
     alsomatch="http://search.yahoo.com/bin/search"
-    update="http://www.mozilla.org/products/firefox/plugins/yahoo.src"
-    updateIcon="http://www.mozilla.org/products/firefox/plugins/yahoo.gif"
-    updateCheckDays="3"
+#   update="http://www.mozilla.org/products/firefox/plugins/yahoo.src"
+#   updateIcon="http://www.mozilla.org/products/firefox/plugins/yahoo.gif"
+#   updateCheckDays="3"
 >
 
 </search>
